### PR TITLE
Alphabetise Cosmos settings in LocalSettings.php

### PR DESCRIPTION
**Pull request by AgentIsai**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -810,7 +748,103 @@
 		'batmanwiki' => true,
 		'snapwikiwiki' => true,
 	],
-
+	'wgCosmosFooterBackgroundColor' => [
+		'default' => '#c0c0c0',
+	],
+	'wgCosmosRailBlacklistedNamespaces' => [
+		'default' => [
+			'NS_SPECIAL',
```
Comment (Universal-Omega) [inline LocalSettings.php:756]: Should without the string, just constants for this and all below. Though we can probably add this to ManageWiki namespaces also.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (AgentIsai) [reply LocalSettings.php]: Removed in favor of https://github.com/miraheze/mw-config/pull/4548

File: LocalSettings.php [hunk 2]
```diff
@@ -810,7 +748,76 @@
 		'batmanwiki' => true,
 		'snapwikiwiki' => true,
 	],
-
+	'wgCosmosFooterBackgroundColor' => [
+		'default' => '#c0c0c0',
+	],
+	'wgCosmosRailBlacklistedNamespaces' => [
```
Comment (Universal-Omega) [inline LocalSettings.php:754]: Defaults are not set in LS for configs in ManageWikiNamespaces.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 2 above)
Comment (AgentIsai) [reply LocalSettings.php]: Removed, thanks!

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**